DDG4/python/DDSim/Helper/Action.py

```diff
@@ -108,3 +122,71 @@ def calorimeterSDTypes(self):
   @calorimeterSDTypes.setter
   def calorimeterSDTypes(self, val):
     self._calorimeterSDTypes = ConfigHelper.makeList(val)
+
+  @staticmethod
+  def makeListOfDictFromJSON(val):
+    if isinstance(val, str):
+      # assumes: valid JSON string or comma-separated list of names
+      import json
+      try:
+        val = json.loads(val)
+      except ValueError:
+        val = tuple(val.split(","))
```

Comment:
Upon reflection, this is still incorrect and should be:
```suggestion
        val = [dict(name=v) for v in val.split(",")]
```